www/js/services.js

```diff
@@ -8,17 +8,15 @@ angular.module('starter.services', [])
         viewAs : 'hymn',
         font : 'gurbaniakhar',
         search : 'gurmukhi_search'
-    }
+    };
 
     if(data.length===0)
     {
-      Store.set(store,defaults);
-      return Store.get(store);
-    }
-    else
-    {
-      return Store.get(store);
+      data = defaults;
```

Comment:
I will not override default as data = defaults only happens when the user do not have any preference set. I have refactor the way we was loading default options. Please see the whole code together and u will see the logic i have written now.